README.md

```diff
@@ -3,29 +3,33 @@
 [![Build Status](https://travis-ci.org/Code-4-Community/backend-scaffold.svg?branch=master)](https://travis-ci.org/Code-4-Community/backend-scaffold)
 [![Coverage Status](https://coveralls.io/repos/github/Code-4-Community/speak-for-the-trees/badge.svg?branch=master)](https://coveralls.io/github/Code-4-Community/speak-for-the-trees?branch=master)
 
-This project is the Java backend api scaffold for Code4Community. It is meant to demonstrate API routing and database connection techniques that should be utilized in all C4C Java back ends.
-
-#### Complete Setup Guide
- - See https://docs.c4cneu.com/getting-started/setup-local-dev/ for a complete setup guide for:
-   - Installing, creating, and running a local PostgreSQL database (`backend-scaffold`)
-   - Configuring IntelliJ
-   - Installing Maven
-   - Installing Java 8
-   - Configuring project properties files
-   - Compiling and running the API
-
-#### Compile the Code Base
- - If `mvn clean install` fails because of `spotless:check`, 
- then run `mvn spotless:apply` to apply code formatting corrections before
- re-running `mvn clean install`. 
-
-#### Run the API
- - The `ServiceMain.java` class has the main method for running the code, this can be run directly in IntelliJ.
- - Alternatively: `mvn install` creates a jar file at:
- `service/target/service-1.0-SNAPSHOT-jar-with-dependencies.jar`.
-   - This can be run from the command line with the command `java -jar service-1.0-SNA....`.
-
-#### Hitting the API
- - By default the API is accessible at `http://localhost:8081`.
- - All routes have the `/api/v1` prefix to them.
- - **Example**: the HTTP request to get all notes would be be: `GET http://localhost:8081/api/v1/notes`.
+This is the Java backend API for the [Speak for the Trees frontend](https://github.com/Code-4-Community/speak-for-the-trees-frontend). 
+
+## Setup :wrench:
+First follow all the steps for setting up the general development environment which can be found [here](https://docs.c4cneu.com/getting-started/setup-local-dev/). This will walk you through:
+- Installing, creating, and running a local PostgreSQL database (`speak-for-the-trees`)
+- Configuring IntelliJ
+- Installing Maven
+- Installing Java 8
+- Configuring project properties files
+- Compiling and running the API
+
+Following these steps, all that is left is to import data into the database. This has been made easy through import routes. You can find these import routes in the Postman collection posted below. Follow the steps below in order to import all the relevant data:
+- Call `POST api/v1/protected/import/neighborhoods` with `persist/src/main/resources/db/seed/neighborhoods.json` as the body
```

Comment:
Before this step you have to create a super admin and user their JWT token. Here's a rough outline of what that step includes:
- Call `POST api/v1/user/signup` to create a super admin user with a JSON body that follows the format of:
```json 
{
    "username": "someUsername",
    "email": "some@email.com",
    "password": "somePassword",
    "firstName": "someFirstName",
    "lastName": "someLastName"
}
```
- Using your preferred method for postgres (Intellij, PgAdmin, postgres consol, ext...) edit the user row you just created so that their 'privilege_level' column is equal to 'SUPER_ADMIN'
- Call `POST /api/v1/user/login` to login to your admin user with a JSON body that follows the format of:
```json
{
    "email": "your@email.com",
    "password": "yourPassword"
}
```
- Copy the "accessToken" returned by that call and add it as the _value_ of a header called 'X-Access-Token' for all seeding API calls